common/sass/fonts/icons/_mixins.scss

```diff
@@ -0,0 +1,17 @@
+@mixin nc-rotate($degrees, $rotation) {
+  filter: progid:DXImageTransform.Microsoft.BasicImage(rotation=#{$rotation});
+  -webkit-transform: rotate($degrees);
+  -moz-transform: rotate($degrees);
+  -ms-transform: rotate($degrees);
+  -o-transform: rotate($degrees);
+  transform: rotate($degrees);
+}
+
+@mixin nc-flip($horiz, $vert, $rotation) {
+  filter: progid:DXImageTransform.Microsoft.BasicImage(rotation=#{$rotation});
+  -webkit-transform: scale($horiz, $vert);
+  -moz-transform: scale($horiz, $vert);
+  -ms-transform: scale($horiz, $vert);
+  -o-transform: scale($horiz, $vert);
+  transform: scale($horiz, $vert);
+}
```

Comment:
You can ditch these mixins, the browsers that need this level of fallback don't support the JS features needed to run the site anyway. And if we do need fallbacks, we should be using something like [cssnext](http://cssnext.io/).

Reply:
yup they aren't necessary, they're just auto-generated mixins I copy pasted [nucleo](https://nucleoapp.com/)